apps/token-manager/contracts/TokenManager.sol

```diff
@@ -303,7 +302,7 @@ contract TokenManager is ITokenController, IForwarder, AragonApp {
         return _spendableBalanceOf(_holder);
     }
 
-    function transferableBalance(address _holder, uint256 _time) public view isInitialized returns (uint256) {
+    function transferableBalance(address _holder, uint64 _time) public view isInitialized returns (uint256) {
```

Comment:
I don't think there is a strong enough reason to break the interface (this function signature changes) of the TokenManager for this change

Reply:
Ahh shoot, that's true :(. Will revert back to using `Uint256Helpers` for this.

Reply:
Ended up just leaving current time as uint256, as we never use it to store a value and only use it comparisons (where leaving it as uint256 is cheaper).